### ISSUE TITLE
What is anchor seed?

### ISSUE THREAD

**Issue by Caliber2** [omni-us/squeezedet-keras#26]
I understand we finally need to give (x,y,w,h) to the network where x,y is grid centre and w,h are width,height of grid. But, in the create_config.py I don't understand how seed is used for calculating anchor boxes. It's being multiplied by (H,W) and reshaped to (H,W,B,2) what is 2 here? Is that 2 really needed? I have to change anchor box dimensions for my dataset. How do I do that?

**Comment by alexandruvesa** [comment]
Yes, same problem . How do i generate the anchor boxes for my own dataset?
Thanks a lot, 
Alex
